### PR TITLE
feat(backup): Send relocation started email

### PR DESCRIPTION
**Pull request by azaslavsky**
Once we have decent confidence that a relocation is not obviously invalid, we send the owner (and creator, if they are different people) an email indicating that their relocation has started.

Screenshot 2023-11-15 at 15 39 35

Screenshot 2023-11-15 at 15 39 43

Issue: getsentry/team-ospo#203

### CODE REVIEW

File: src/sentry/templates/sentry/debug/mail/preview.html [hunk 1]
```diff
@@ -53,6 +53,9 @@
         <option value="mail/sentry-app-notify-disable/">Sentry App Notify Disable</option>
         <option value="mail/join-request/">Join Request</option>
       </optgroup>
+      <optgroup label="Relocation">
+        <option value="mail/relocation-started/">Relocation Started</option>
```
Comment (hubertdeng123) [inline src/sentry/templates/sentry/debug/mail/preview.html:57]: should we move `Relocate Account` here if this is now a new group?

File: src/sentry/templates/sentry/emails/relocation_started.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{% extends "sentry/emails/base.html" %}
+
+{% load i18n %}
+
+{% block main %}
+    <h3>Relocation Accepted</h3>
+    <p>Your relocation (id: <i>{{ uuid }}</i>) request has been accepted. You requested that the following organizations be migrating to sentry.io:</p>
```
Comment (hubertdeng123) [inline src/sentry/templates/sentry/emails/relocation_started.html:7]: It feels a bit weird to have the uuid in parentheses here, would it be better to have that in smaller text underneath the header `Relocation Accepted`? Not a designer though, so maybe I don't know what I'm talking about

File: src/sentry/templates/sentry/debug/mail/preview.html (hunk 1 above)
Comment (azaslavsky) [reply src/sentry/templates/sentry/debug/mail/preview.html:57]: Done.

File: src/sentry/templates/sentry/emails/relocation_started.html (hunk 2 above)
Comment (azaslavsky) [reply src/sentry/templates/sentry/emails/relocation_started.html:7]: Done.

I've updated the images attached to the PR to reflect this.

**Review by hubertdeng123 — APPROVED** [review]

**Pull request merged**